src/.vuepress/enhanceApp.js

```diff
@@ -80,29 +86,107 @@ const getSidebar = version => {
   /* OPERATIONS SECTION START */
   const operations = {
     title: 'Operations',
-    collapsable: false,
+    collapsable: true,
     sidebarDepth: 0,
     children: [
       `${version}/operations/planning`,
       `${version}/operations/service`,
       `${version}/operations/monitoring`,
     ]
   };
-
-  if (minVersion('1.2.3')) {
-    operations.children.push(`${version}/operations/backwards-compatibility`);
+  if (minVersion('1.1.0') && maxVersion('1.2.4')) {
+    operations.children.push(`${version}/develop/auditor`);
```

Comment:
This is duplicated in https://github.com/codenotary/immudb.io/pull/161/files#diff-dd81526d762c2a0f0d03562e6c13763b34f916e25c5be345cf4e0282e2731923R218 - previously it was only in the `develop` section